package/MDAnalysis/coordinates/base.py

```diff
@@ -1741,6 +1746,49 @@ def get_aux_descriptions(self, auxnames=None):
             auxnames = self.aux_list
         descriptions = [self._auxs[aux].get_description() for aux in auxnames]
         return descriptions
+    
+    def add_transformation(self, transform, args=None):
+        """ Add a transformation to be applied to the trajectory
+        
+        Documentation will be added here
+        
+        See Also
+        --------
+        :meth:`remove_transformation`
+        """
+        self._transforms[transform] = args
+        self.ts = transform(self.ts, self._transforms[transform])
+        # call reader here to apply the newly added transformation on the
+        # current loaded frame?
+    
+    def remove_transformation(self, transform):
+        """ Remove a transformation from the workflow
+        Documentation will be added here
+        
+        See Also
+        --------
+        :meth:`add_trasnformation`
+        """
+        try:
+             self._check_for_transformation(transform)
+        except:
+            raise ValueError("No transformation named {name}".format(name=transform))
```

Comment:
You are not referring to transformations by name. The error message is confusing.